pkg/domain/infra/abi/play.go

```diff
@@ -299,6 +299,18 @@ func (ic *ContainerEngine) playKubePod(ctx context.Context, podName string, podY
 		return nil, err
 	}
 
+	var ctrRestartPolicy string
+	switch podYAML.Spec.RestartPolicy {
+	case v1.RestartPolicyAlways:
+		ctrRestartPolicy = libpod.RestartPolicyAlways
+	case v1.RestartPolicyOnFailure:
+		ctrRestartPolicy = libpod.RestartPolicyOnFailure
+	case v1.RestartPolicyNever:
+		ctrRestartPolicy = libpod.RestartPolicyNo
+	default: // Default to Always
+		ctrRestartPolicy = libpod.RestartPolicyAlways
```

Comment:
Is this the default? I remember the upstream issue mentioning OnFailure was the default for Kube. @haircommander Would you happen to know this?

Reply:
```
[root@k8s-m1 ~]# kubectl explain pod.spec.restartPolicy
KIND:     Pod
VERSION:  v1

FIELD:    restartPolicy <string>

DESCRIPTION:
     Restart policy for all containers within the pod. One of Always, OnFailure,
     Never. Default to Always. More info:
     https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#restart-policy
```

Reply:
@mheon, if you are referring to my issue, then I took this info from the [this doc](http://docs.podman.io/en/latest/markdown/podman-generate-systemd.1.html):
```
–restart-policy=policy

Set the systemd restart policy. The restart-policy must be one of: “no”, “on-success”, “on-failure”, “on-abnormal”, “on-watchdog”, “on-abort”, or “always”. The default policy is on-failure.
```

But now I see that it's a wrong doc. In Kubernetes the default restart policy is indeed `Always`.